lib/lotus/helpers/html_helper/empty_html_node.rb

```diff
@@ -12,6 +12,12 @@ class EmptyHtmlNode
         # @api private
         ATTRIBUTES_SEPARATOR = ' '.freeze
 
+        # HTML5 form elements where required should be reformated
+        #
+        # @since 0.2.1
+        # @api private
+        HTML5_REQUIRED_CONSTRAINT = %('input' 'select' 'textarea' 'radio').freeze
```

Comment:
naming is hard, what do you think about having a hash instead here and also pluralize the naming:

```
HTML5_REQUIRED_CONSTRAINTS = {
  'required' => %('input' 'select' 'textarea' 'radio')
}.freeze
```

This paves way for further addition of new contraints

Furthermore, please update the comment accordingly

After this, it is :+1: to merge